### ISSUE TITLE
Problem with CSharpLexerBase

### ISSUE THREAD

**Issue by ferasfares** [antlr/grammars-v4#2611]
Describe the bug  
Generate CSharpLexer.py using antlr4 version 4.10.1 
try to import the file 
 File "CSharpLexer.py", line 12, in <module>
    from .CSharpLexerBase import CSharpLexerBase
ModuleNotFoundError: No module named 'grammar.CSharp.CSharpLexerBase'

in the repository I can't find class CSharpLexerBase.py or CSharpLexerBase.g4 to generate class 

thanks

**Comment by ferasfares** [comment]
Another problem in generated code I have some thing like that in CSharpLexer.py 
```
def INTERPOLATED_VERBATIUM_STRING_START_action(self, localctx:RuleContext , actionIndex:int):
        if actionIndex == 1:
             this.OnInterpolatedVerbatiumStringStart(); 
```

**Comment by kaby76** [comment]
Right. There's no grammars-v4/csharp/Python3/ directory. In other words, there's no Python3 implementation for ]the grammar](https://github.com/antlr/grammars-v4/tree/master/csharp). There's a bit of controversy over "target agnostic format", which is why I hadn't implemented the Python3 base target classes.

**Comment by abulka** [comment]
So is there a solution? 

I once (3-4 years ago) got an earlier version of the CSharp grammar working in Python 3 but now I need the latest grammar which hopefully supports generics syntax etc.
